torch/_inductor/codegen/cpp.py

```diff
@@ -1948,10 +2794,14 @@ def run(kernel):
                     inner_most_idx - outer_tiling_idx, factor=tiling_factor
                 )
                 inner_main_loop.set_kernel(
-                    codegen_kernel(CppTile2DKernel, tiling_factor, outer_tiling_idx)
+                    codegen_kernel(
+                        CppTileKernel,
+                        [tiling_factor, tiling_factor],
+                        [outer_tiling_idx, inner_most_idx],
+                    )
                 )
                 inner_tail_loop.set_kernel(
-                    codegen_kernel(CppTile2DTailKernel, tiling_factor, outer_tiling_idx)
+                    codegen_kernel(CppTileKernel, [tiling_factor], [outer_tiling_idx])
```

Comment:
`CppTile2DKernel` and `CppTile2DTailKernel` are not used any longer. Will remove them in the next PR to make the change simpler in this PR.